lib/knapsack/adapters/minitest_adapter.rb

```diff
@@ -46,8 +46,17 @@ def set_test_helper_path(file_path)
         @@parent_of_test_dir = File.expand_path('../', test_dir_path)
       end
 
+      def add_post_run_callback(&block)
+        if Minitest.respond_to?(:after_run)
+          Minitest.after_run { block.call }
+        else
+          Minitest::Unit.after_tests { block.call }
+        end
+      end
+
       def self.test_path(obj)
-        test_method_name = obj.name
```

Comment:
You experienced problem with this in case when you had helper methods without `test_` in name?